core/src/main/java/org/apache/carbondata/core/constants/CarbonCommonConstants.java

```diff
@@ -1459,6 +1459,13 @@
    */
   public static final String HANDOFF_SIZE = "carbon.streaming.segment.max.size";
 
+  /**
+   * enable auto handoff streaming segment
+   */
+  public static final String ENABLE_AUTO_HANDOFF = "carbon.enable.auto.handoff";
```

Comment:
I think `carbon.streaming.auto.handoff.enabled` is better.

Reply:
ok